src/D2L.Security.OAuth2/Keys/Default/RotatingPrivateKeyProvider.cs

```diff
@@ -13,8 +12,16 @@ internal sealed class RotatingPrivateKeyProvider : IPrivateKeyProvider {
 		private readonly IDateTimeProvider m_dateTimeProvider;
 		private readonly TimeSpan m_keyRotationPeriod;
 
+		[Mutability.Audited(
+			"Todd Lang",
+			"14-Aug-2018",
+			".Net class we can't mark, used internally without mutability." )]

```

Comment:
Unlike `CngAlgorithm`, I think this one makes sense to audit.  We don't want to automatically assume that every SemaphoreSlim is safe.  We should probably still manually inspect these to make sure that they are still sane in a multi-tenant environment.